idea/src/org/jetbrains/kotlin/idea/actions/NewKotlinFileAction.java

```diff
@@ -55,8 +55,8 @@ protected void postProcess(PsiFile createdElement, String templateName, Map<Stri
     @Override
     protected void buildDialog(Project project, PsiDirectory directory, CreateFileFromTemplateDialog.Builder builder) {
         builder
-                .setTitle(JetBundle.message("new.kotlin.file.action"))
-                .addKind("Kotlin file", JetFileType.INSTANCE.getIcon(), "Kotlin File")
+                .setTitle("New Kotlin File/Class")
```

Comment:
This is the dialog title, not the name of the action in the menu. The dialog titles for Java and Groovy do have "New" in them.